src/matches.ts

```diff
@@ -0,0 +1,58 @@
+import { purry } from './purry';
+
+function _matches(target: any, condition: any) {
```

Comment:
Let's use `unknown` here, I'm planning on turning on stricter typescript and we won't be able to use `any` anywhere
```suggestion
function _matches(target: unknown, condition: unknown) {
```

Reply:
That breaks.